content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -30,6 +30,20 @@ Specifically, Odoo's predictive lead scoring uses the *naive Bayes* probability
    P(A | B) = \frac{P(A) \times P(B | A)}{P(B)}
    \end{equation}
 
+Breaking down the equation:
+
+- P(A|B) = The probability of a successful lead *in this case*
+- P(A) = The overall probability of a lead being successful regardless of the conditions
+- P(B|A) = The probability of this being the case given a lead is successful
+- P(B) = The probability of this being the case
```

Comment:
Suggesting to use the :math: role here, so that the formatting matches the equation. Also, suggesting to use colons instead of "=" just to avoid any confusion of the equals being taken as part of the definition :)
```suggestion
- :math:`P(A|B)`: The probability of a successful lead *in this case*
- :math:`P(A)`: The overall probability of a lead being successful regardless of the conditions
- :math:`P(B|A)`: The probability of this being the case given a lead is successful
- :math:`P(B)`: The probability of this being the case
```